### PR TITLE
addition of an armor set with 2 color variants with special craftable variant and a new RIG suit for suit up

### PR DESCRIPTION
**Pull request by Karlov357**
## About The Pull Request
<details>
<summary>
	addition of an armor set with 2 color variants with special craftable variant and a new RIG suit for suit up
</summary>
<hr>

	Addition of a new mid tier armor suit for cargo's Sol Fed station since all armors are basic vests from starter stations or the assault suit from the kriosians. The new "sol fed" armor seeks to bridge the middle area between simple vest and end tier assault armor, simply balanced by putting it in a far (arguably 1 hour unlock) station while being affordable to the standard colonist and also adding a special variant which sacrifices modification of armor values with the ability to attach the chest rig craftable to give the armor a 5 cell internal inventory. The RIG suit seeks to add a new (slightly more expensive) light suit which offers high mobility and decent protection compared to heavier combat RIGs (low speed loss and slightly better armor than standard EVA RIG, similar to light RIG but atmos friendly). This adds new mob and icon sprites to accommodate all variants and pieces of the armors and RIGs, the respective changes to the armor code pages, and the changes to illegalstation1 and suitup station code pages to add both sets of armor to their intended stations. (note that the solfed armor price may need to be reviewed as a proper price point was hard to determine).

<hr>
</details>

## Changelog
:cl:

/:cl:

### CODE REVIEW

File: code/modules/clothing/spacesuits/rig/suits/station.dm [hunk 1]
```diff
@@ -519,3 +519,46 @@ Technomancer RIG
...
+/***************************************
+	MU66 Suit
+****************************************/
+/obj/item/rig/sea
+	name = "MU-66 Atmos suit"
+	suit_type = "MU66 Soft-suit"
+	desc = "A light RIG for performing minute repairs and maintenance in atypical atmospheres."
+	icon_state = "sea_rig"
+	armor_list = list(
+		melee = 40,
```
Comment (benj8560) [inline code/modules/clothing/spacesuits/rig/suits/station.dm:532]: these are numbers for the old armor system. This suit will make you near indestructible please look at the other suits for examples.

Comment (benj8560) [reply code/modules/clothing/spacesuits/rig/suits/station.dm:532]: infact same goes for **all** the armors you have added

File: code/modules/clothing/spacesuits/rig/suits/station.dm [hunk 2]
```diff
@@ -519,3 +519,46 @@ Technomancer RIG
...
+		bomb = 10,
+		bio = 100,
+		rad = 100
+	)
+	seal_delay = 20
+	slowdown = 0
+	offline_slowdown = 1
+	offline_vision_restriction = 1
+	removable_cell = TRUE
+	cell_type =  /obj/item/cell/large/moebius/nuclear
```
Comment (benj8560) [inline code/modules/clothing/spacesuits/rig/suits/station.dm:544]: suit includes a large atom cell...? Seems kinda bad for balance

Comment (Karlov357) [reply code/modules/clothing/spacesuits/rig/suits/station.dm:544]: man, I kinda overlooked the armor things on accident. I forgot I made this before the new armor PR and just wanted to rush it through to get things off my plate. I'll get the issues changed up real quick

**Comment by Karlov357** [comment]
the corrections have been made, still a bit new though to the PR game. The new commit is attached here but if I need to close and restart a new PR just let me know

**Comment by benj8560** [comment]
everything appears functionally sound. Balance seems...well these vests seem kinda _insaine_ to me but up to whoever's merging it I guess.  these are pretty much assault armor tier.

**Comment by cdb-is-not-good** [comment]
This needs a serious balance pass.

**Comment by Karlov357** [comment]
> everything appears functionally sound. Balance seems...well these vests seem kinda _insaine_ to me but up to whoever's merging it I guess. these are pretty much assault armor tier.

The values of the armor are exactly between assault armor and a standard vest, which is what was meant by bridging the gap since it’s a mid tier armor (standard vest being low tier and assault being high/end tier). Not only that this new armor will functionally take about as long if not longer to acquire through cargo than the assault vest as cargo can purchase an assault suit essentially off the rip within 5 minutes of the game while arau station takes roughly an hour or more depending on rng for the standard CT.

**Comment by Karlov357** [comment]
Currently talking with Trilby about comparable armors and how to better balance the values since I had an initial misunderstanding of how the values actually scale.

**Pull request closed without merging**